test/test-utils/client.ts

```diff
@@ -168,5 +168,6 @@ export const mockClientMethodsCrypto = (): Partial<
         isCrossSigningReady: jest.fn().mockResolvedValue(true),
         isSecretStorageReady: jest.fn(),
         getSessionBackupPrivateKey: jest.fn(),
+        getVersion: jest.fn().mockReturnValue("Version 0"),
```

Comment:
is there a mock of `olmVersion` somewhere which we can get rid of?

Reply:
There is no mock of `olmVersion`, the value was directly read on the `MatrixClient`, so `undefined` in the test